### DISCUSSION TITLE
[元气骑士官方入驻][委托]乐队的应援粉丝团，集合！  前往地牢深处，为喜欢的乐队举灯！

### DISCUSSION THREAD

**Discussion by tianmu666** [share121/inter-knot#858] [其他]
![1723283225444](https://github.com/user-attachments/assets/3510a05e-f6b7-4a34-ac3c-01717ba39faa)
感兴趣的绳匠可前往b站官号提交委托

**Comment by flowfwa** [comment]
尊嘟假嘟

**Reply by tianmu666** [reply]
真的，不信去看B站官号

**Comment by lingyushen** [comment]
肉鸽，好玩嘛

**Reply by tianmu666** [reply]
当然好玩，这游戏的精华是各种各样“特性”，以肉鸽元素为辅，还有……瑟瑟的立绘？……锦上添花？……（不是[狗头]）

**Comment by bugmaibear** [comment]
逆天，怎么哪哪都有元（狗头）
